src/Stratis.Bitcoin/Utilities/RewindData.cs

```diff
@@ -60,6 +60,9 @@ public List<UnspentOutputs> OutputsToRestore
             }
         }
 
+        /// <summary>The size of the block in bytes, the block must be serialized for this property to be set.</summary>
+        public long? BlockSize { get; protected set; }
```

Comment:
no need to save it. 

I see you need it in cached coinview for batching. Instead create a class which will have 2 members- rewind data and block size (or maybe some more metadata if you need) and use that class with metadata in the async queue. 
But don't save block size to the DB, we don't need it